crowbar_framework/config/boot.rb

```diff
@@ -55,7 +55,7 @@
   gem "dotenv", version: "~> 1.0"
   require "dotenv"
 
-  gem "hashie", version: "~> 2.1"
+  gem "hashie", version: "~> 3.4"
```

Comment:
In another PR you suggested to use patch level versioning. Still makes sense here?